### ISSUE TITLE
Notifications / PubSub after erasing & recreating instance

### ISSUE THREAD

**Issue by qsypoq** [tootsuite/mastodon#2651]
Hi, so, this is what happened (TL;DR at the end) : 

I deleted all my mastodon related files, except the docker-compose & nginx configuration by being a massive noob. So, no more database & stuff.

Repulled everythings, reinstalled everythings like before the accident, keeped the same @ & subdomain.

Before, all was working wonderfully, then, after my noobish rm -rf : 
- I'm still listed on the following list of the people who were following me before.
- If they check my profile, they sees me as non-following.
- If i follow someone, they don't get the notification, so i tested to follow an alt account i wasn't following before : still no notification. (But i was following people on the instance)
- They don't get notification for mention nor can see my toots & replies
- My PubSubHubbub list is empty
- On others instance admin's panel my old PubSub entry is still here
- If i follow someone on a instance were i was not "federated" before the apocalypse they do get the notification and the PubSubHubbub entry will be added and they can see my toots

What i think : 
- Maybe the fact that i already exist on some instance made the PubSub to ignore the "new" one
- Maybe after the renewall of the PubSubHubbub list everything will be fine and i'm wasting our times
- Maybe not, this is why i'm reporting this.

TL;DR : 
- Deleted my instance, recreated it with same @ & subdomain
- All instances connected before : Can't get my toots/replies/notifications, ignore PubSub
- All non-connected instances before : Work fine

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by Gargron** [comment]
Webfinger is cached for a day so some things might start working in due time again, like new follow notifications and stuff. But old data about your old accounts on other instances is there to stay.

**Comment by qsypoq** [comment]
I also get a lot of these : 
```

ERROR:  duplicate key value violates unique constraint "index_accounts_on_username_and_domain"
DETAIL:  Key (username, domain)=(NICKNAME, INSTANCE) already exists.
STATEMENT:  INSERT INTO "accounts" ("username", "domain", "public_key", "remote_url", "salmon_url", "hub_url", "created_
at", "updated_at", "uri", "url", "last_webfingered_at") VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11) RETURNING
"id"
```
